From 9966e10f36876daa910ce301a175633771470e82 Mon Sep 17 00:00:00 2001
From: Samuli Piippo <samuli.piippo@theqtcompany.com>
Date: Wed, 18 Nov 2015 10:14:18 +0200
Subject: [PATCH] Remove qgltf

assimp does not compile for ARM because of GCC bug
(https://gcc.gnu.org/bugzilla/show_bug.cgi?id=66964)

Task-number: QTBUG-49153
---
 tools/tools.pro | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tools/tools.pro b/tools/tools.pro
index 09c7d8e..8f093ce 100644
--- a/tools/tools.pro
+++ b/tools/tools.pro
@@ -1,3 +1,3 @@
 TEMPLATE = subdirs
-!android:SUBDIRS += qgltf
+#!android:SUBDIRS += qgltf
 qgltf.CONFIG += host_build
